core-bundle/contao/templates/twig/content_element/description_list.html.twig

```diff
@@ -0,0 +1,14 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{% block content %}
+    <dl>
+        {% for description in descriptions %}
+            {% if description.term is not empty %}
+                <dt>{{ description.term|insert_tag_raw }}</dt>
+            {% endif %}
+            {% if description.details is not empty %}
+                <dd>{{ description.details|insert_tag_raw }}</dd>
+            {% endif %}
+        {% endfor %}
+    </dl>
```

Comment:
I would add attributes to the three main tags as well as a block that allows grouping the description (e.g. in a div) as described here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl?retiredLocale=de#wrapping_name-value_groups_in_div_elements.

```suggestion
    <dl{{attrs(description_list_attributes|default)}}>
        {% for description in descriptions %}
            {% block description %}
                {% if description.term is not empty %}
                    <dt{{attrs(description_term_attributes|default)}}>{{ description.term|insert_tag_raw }}</dt>
                {% endif %}
                {% if description.details is not empty %}
                    <dd{{attrs(description_details_attributes|default)}}>{{ description.details|insert_tag_raw }}</dd>
                {% endif %}
            {% endblock %}
        {% endfor %}
    </dl>
```